docker/Dockerfile

```diff
@@ -17,12 +17,7 @@ FROM python:3.7-slim as builder
 # (we really just care about caching a wheel here, as the "pip install" below
 # will install them again.)
 
-# we need dependencies for postgresql
-RUN apt-get update && apt-get install -y gcc git libpq-dev
-
-RUN pip install --prefix="/install" --no-warn-script-location cryptography
```

Comment:
Hm, do we really not need cryptography anymore?

Reply:
Also, still need to remove the installation of `libpq` on line 36/31 (underneathe 'Stage 1: runtime') :)

Reply:
So if what you mean by "it's been tried" is that the tests pass with this configuration, then the answer is yes :) Also I can't remember but I'm pretty sure I did a fresh install of Sygnal in a new directory with this configuration to make sure that the installation/build process went fine, but someone might want to double check that on a different machine.

Reply:
> Still need to remove the installation of `libpq` on line 36/31 (underneathe 'Stage 1: runtime') :)

It's still there (unless I'm misunderstanding GitHub's review system)

We won't need `libpq5` at runtime if there are no databases involved :D